patches/001_email-verified.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/lib/profileMapper.js b/lib/profileMapper.js
+index 31ef24a..b503e97 100644
+--- a/lib/profileMapper.js
++++ b/lib/profileMapper.js
+@@ -1,4 +1,13 @@
+ module.exports = function (raw_data) {
++  /* Mozilla LDAP specific aliases handling */
++  var all_emails = [];
++  if (raw_data.mail) {
++    all_emails = all_emails.concat(raw_data.mail);
++  }
++  if (raw_data['zimbraAlias']) {
++    all_emails = all_emails.concat(raw_data['zimbraAlias']);
++  }
++
+   var profile = {
+     id:          raw_data.objectGUID || raw_data.uid || raw_data.cn,
+     displayName: raw_data.displayName,
+@@ -8,7 +17,7 @@ module.exports = function (raw_data) {
+     },
+     nickname: raw_data['sAMAccountName'] || raw_data['cn'] || raw_data['commonName'],
```

Comment:
This code is mixing dot and bracket notation. Use only dot notation (`raw_data.sAMAccountName`) unless looking up by variable.

Reply:
This is Auth0's code which I'd like to keep intact and change as little as possible. To improve this we'd want to PR to their upstream : https://github.com/auth0/ad-ldap-connector/